pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -63,8 +63,12 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		return response.JSON(http.StatusInternalServerError, ruleResponse)
 	}
 
-	for _, groupId := range ruleGroupQuery.Result {
-		alertRuleQuery := ngmodels.ListRuleGroupAlertRulesQuery{OrgID: c.SignedInUser.OrgId, RuleGroup: groupId}
+	for _, r := range ruleGroupQuery.Result {
+		if len(r) < 3 {
```

Comment:
@papagian is it possible to ever have this be true? All three fields are not nullable so shouldn't we always have all three fields set in any result?

Reply:
I think it's not possible but I've added just to be safe. Do you think it's better to remove it?

Reply:
No I think it's fine, just checking that I understand what's going on correctly. The check is sensible and about as light weight as possible.